### DISCUSSION TITLE
Configure reporter from the environment variable

### DISCUSSION THREAD

**Discussion by nijel** [biomejs/biome#3474] [Ideas]
Presently, it is possible to configure reporter on the command line only. I'd like to be able to configure this using an environment variable as well.

We use biome in pre-commit. When this is triggered by the user, we want nicely formatted output (what is the default). But when running in GitHub CI, I'd like to have `github` reporter so that it can be parsed by GitHub.

With ruff, we achieve the same by setting `RUFF_OUTPUT_FORMAT=github` in the CI environment, and having something similar for biome would be great.

**Comment by ematipico** [comment]
That's a nice addition!
